lib/server.js

```diff
@@ -386,6 +406,88 @@ module.exports = function createServer(config, log) {
     }
   )
 
+  api.post('/checkCustomRule', function(req, res, next) {
```

Comment:
nit: this api still says "custom rule", which no longer matches the terminology used in the code here.

Reply:
By moving this into its own route, we bypass all the checks in the usual `/check` route.  IIUC that's mostly what we want, but I wonder if we've lost any that we should keep, such a the ip-reputation-service checking, the ip-blocklist checking, or the explicit "block this IP address" API call offered by the `/blockIp` route?